auto_round/wrapper.py

```diff
@@ -167,24 +168,39 @@ def _qdq_weight(self, value, min_scale, max_scale):
         if isinstance(self.orig_layer, transformers.modeling_utils.Conv1D):
             weight = weight.t()
 
-        weight_q, scale, zp = self.weight_quant_func(
-            weight,
-            bits=self.orig_layer.bits,
-            group_size=self.orig_layer.group_size,
-            v=value,
-            min_scale=min_scale,
-            max_scale=max_scale,
-            scale_dtype=self.orig_layer.scale_dtype,
-            tensor_min=self.weight_min,
-            tensor_max=self.weight_max,
-            data_type=self.data_type,
-            q_scale_thresh=self.q_scale_thresh,
-
-        )
+        if self.dq:
```

Comment:
change the return values to weight_q, (scale,d_scale), (min,d_min), thus could call the same function